content/docs/distributed-asynchronous.md

```diff
@@ -0,0 +1,51 @@
+# Message delivery patterns in asynchronous systems
+
+First of all we should ask ourselves a question: why both with distributed asynchronous systems? Wouldn't it be better to just build synchronous non-distributed ones? It certainly would be easier.
+
+### Why distributed
+
+A system is *distributed* if it consists of multiple components and these components are running inside different processes. The fact that these components are part of a single system means they need to communicate to realise functions of that systems. If there was no requirement for communication then we could call them separate non-distributed systems.
+
+Communication across process boundaries is much more difficult than within a single process. A call to another service needs to be converted into a message that can be transmitted over the wire. On the other side a similar mechanism needs to unpack the message and convert it back into a function call. There are many things that can go wrong in the meantime, including but not limited to network issues and process downtime.
```

Comment:
I would use a different word than "difficult" maybe "challenging". Difficult might indicate that it's hard at the code level which isn't the biggest challenge.